packages/zostso/README.md

```diff
@@ -7,30 +7,18 @@ Contains APIs to interact with TSO on z/OS (using z/OSMF TSO REST endpoints).
 **Issue the TSO command "status" to display info about jobs for your user ID**
 
 ```typescript
-import { IProfile, Session, Logger, LoggingConfigurer, ImperativeError,
-         CredentialManagerFactory } from "@zowe/imperative";
-import { ZosmfSession } from "@zowe/zosmf-for-zowe-sdk";
-import { getDefaultProfile } from "@zowe/core-for-zowe-sdk";
+import { ProfileInfo } from "@zowe/imperative";
 import { IssueTso } from "@zowe/zos-tso-for-zowe-sdk";
 
 (async () => {
-    //Initialize the Imperative Credential Manager Factory and Logger
-    Logger.initLogger(LoggingConfigurer.configureLogger('lib', {name: 'test'}));
-    // Uncommment the below line if the Secure Credential Store is in use
-    // await CredentialManagerFactory.initialize({service: "Zowe-Plugin"});
-
-    // Get the default z/OSMF profile and create a z/OSMF session with it
-    let defaultZosmfProfile: IProfile;
-    let defaultTsoProfile: IProfile;
-    try {
-        defaultZosmfProfile = await getDefaultProfile("zosmf", true);
-        defaultTsoProfile = await getDefaultProfile("tso", false);
-    } catch (err) {
-        throw new ImperativeError({msg: "Failed to get a profile."});
-    }
-
-    const session: Session = ZosmfSession.createBasicZosmfSession(defaultZosmfProfile);
-    const accountNumber = defaultTsoProfile.account;
```

Comment:
I stopped loading the TSO account number from a profile because the code to do is significantly more complex using the ProfileInfo API:
```typescript
const tsoProfAttrs = profInfo.getDefaultProfile("tso");
const tsoMergedArgs = profInfo.mergeArgsForProfile(tsoProfAttrs);
const accountNumber = tsoMergedArgs.knownArgs.find(arg => arg.argName === "account").argValue as string;
```

Curious what your thoughts are on this. Should I include the above code block in the example, or leave it out? Alternatively, is there a simpler way I've missed to load the account number?

Reply:
Maybe provide two examples? I can't really think of any other alternative.

Reply:
Do we think that getting the TSO account is a common use-case for consumers of this API. If so, then we probably should show them how to accomplish it, even if it is involved. Maybe include a comment in the sample stating that getting the account is not a common practice?